minio/minio
# add '.healing.bin' for tracking currently healing disk

## Description
add '.healing.bin' for tracking currently healing disk

## Motivation and Context
add a hint on the disk to allow for tracking fresh disk
being healed, to allow for restartable heals, and also
use this as a way to track and remove disks.

There are more pending changes where we should move
all the disk formatting logic to backend drives, this
PR doesn't deal with this refactor instead makes it
easier to track healing for now.

## How to test this PR?
Replace drives, let the drive healing start at that point attempt
a restart of the same server, if the drive is not healed yet -
then the drive healing would start again to make sure the
drive is healed properly. 

This PR attempts to add a healing tracker for the drive that
is being healed at the moment. 

## Types of changes

## Checklist:
